src/CLI/actioner/sonic-cli-show-techsupport.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python
+import sys
+import time
+import json
+import ast
+import re
+from scripts.render_cli import show_cli_output
+from rpipe_utils import pipestr
+import cli_client as cc
+import urllib3
+
+api = cc.ApiClient()
+urllib3.disable_warnings()
+plugins = dict()
+
+
+def register(func):
+    """Register sdk client method as a plug-in"""
+    plugins[func.__name__] = func
+    return func
+
+def call_method(name, args):
+    method = plugins[name]
+    return method(args)
+
+def invoke(func, args):
+    body = None
+    aa = cc.ApiClient()
+
+    # Gather tech support information into a compressed file
+    if func == 'rpc_sonic_show_techsupport_sonic_show_techsupport_info':
+        keypath = cc.Path('/restconf/operations/sonic-show-techsupport:sonic-show-techsupport-info')
+        if args is None:
+            body = {"sonic-show-techsupport:input":{"date": None}}
+        else:
+            body = {"sonic-show-techsupport:input":{"date": args[0]}}
+        return aa.post(keypath, body)
+    else:
+        print("%Error: not implemented")
+        exit(1)
+
+
+def run(func, args):
+
+    try:
+        api_response = invoke(func, args)
+
+        if api_response.ok():
+            response = api_response.content
+
+            if response is None:
+                print ("ERROR: No output file generated: \n"
+                       "Invalid input: Incorrect DateTime format")
```

Comment:
The failure may not be due to incorrect date time format always.

Reply:
You're right. Actually it turns out this is never the result of an incorrect Date/time format. This print statement was coded during testing of the "invalid Date/time" case when this failure coincidentally occurred due to a completely different cause. 

I have corrected this message to eliminate the mention of "Incorrect DateTime format". That error is diagnosed separately and has a separate error message in this file.